src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -120,14 +119,14 @@ public virtual void WriteMessage(LogLevel logLevel, string logName, int eventId,
                 logLevelColors = GetLogLevelConsoleColors(logLevel);
                 logLevelString = GetLogLevelString(logLevel);
                 // category and event id
-                logIdentifier = _loglevelPadding + logName + "[" + eventId + "]";
+                sb.AppendLine(_loglevelPadding + logName + "[" + eventId + "]");
                 // scope information
                 if (IncludeScopes)
                 {
-                    scopeInformation = GetScopeInformation();
+                    sb.AppendLine(GetScopeInformation());
                 }
                 // message
-                message = _messagePadding + ReplaceMessageNewLinesWithPadding(message);
+                sb.AppendLine(_messagePadding + ReplaceMessageNewLinesWithPadding(message));
```

Comment:
This is another opportunity to remove allocation, since we do a `string.replace` on the message